src/expandProperty.js

```diff
@@ -67,6 +68,15 @@ function parseCircular(value, resolve) {
   }
 }
 
+function parsePosition(value, resolve) {
+  const [Start, End = Start] = splitShorthand(value)
+
+  return {
+    [resolve('Start')]: Start,
+    [resolve('End')]: End,
+  }
+}
```

Comment:
This is added to expand `{margin|padding}-{inline|block}` into `{margin|padding}-{inline|block}-{start|end}`